include/openenclave/bits/report.h

```diff
@@ -205,6 +206,20 @@ typedef struct _oe_policy
     size_t policy_size;
 } oe_policy_t;
 
+/**
```

Comment:
Per the arch meeting, could you move this function out into `include/openenclave/attestation/sgx/report.h`? 

@yentsanglee, let me know if that's reasonable; I expected that endorsements.h would likely also be made public from that location. 

This would also require an update to include/CMakeLists.txt to publish the new header:
```txt
install(FILES openenclave/attestation/sgx/report.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/openenclave/ COMPONENT OEHOSTVERIFY)
```

Reply:
For this function `oe_get_signer_id_from_public_key()`, it makes sense to reside in `include/openenclave/attestation/sgx`. What happen to other non-attestation helpers? I am thinking that all helper header files should be under the same folder. For example, `include/openenclave/tee/sgx` ?

Reply:
@yentsanglee I was originally thinking maybe just `include/openenclave/sgx` but I think given the precedent set by attestation, we should consider TEE-specific helper libraries on a _functional_ basis (e.g. `include/openenclave/debug/sgx` if we want to produce TEE-specific helper libraries for devs writing debuggers, in the same way that `include/openenclave/attestation/sgx` will be for helpers for the attestation plugins.)

Reply:
Sounds good.